torch/_dynamo/utils.py

```diff
@@ -498,6 +504,23 @@ def clear(self):
         self.values.clear()
 
 
+@overload
+def istype(obj: object, allowed_types: Type[T]) -> TypeGuard[T]:
```

Comment:
also should probably have used https://typing-extensions.readthedocs.io/en/latest/#typing_extensions.TypeIs